doc/tutorials/using_hybrid_mode_on_nuc.rst

```diff
@@ -143,6 +162,15 @@ Perform the following to update Ubuntu GRUB so it can boot the hypervisor and lo
 
 Hybrid Scenario Startup Check
 *****************************
+#. install plug picocom
```

Comment:
We should simply this and instruct the user to set-up the serial port connection. We also have a guide for that we can reference.